src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -485,7 +485,7 @@ public virtual bool SaveAs(string newPath, ProtoCore.Core core)
         {
             if (String.IsNullOrEmpty(newPath)) return false;
 
-            Log("Saving " + newPath + "...");
+            Log(Properties.Resources.SavingInProgress + " " + newPath + "...");
```

Comment:
This is unusual, try `string.Format(Properties.Resources.SavingInProgress, newPath)` instead.